pkg/mgrconfig/load.go

```diff
@@ -234,9 +237,19 @@ func (cfg *Config) completeServices() error {
 	return nil
 }
 
-func (cfg *Config) initTimeouts() {
+func (cfg *Config) initTimeouts() error {
 	slowdown := 1
 	switch {
+	case cfg.Type == "qemu":
```

Comment:
I think this breaks the next case.
Only 1 case from switch is executed, if we already matched `case cfg.Type == "qemu"`, then it's executed, and we never execute `case cfg.Type == "qemu" && runtime.GOARCH != cfg.SysTarget.Arch && runtime.GOARCH != cfg.SysTarget.VMArch:` case.